README.md

```diff
@@ -58,9 +58,13 @@ python3 ./manim.py example_scenes.py SquareToCircle -pl
 
 ## Anaconda Install
 
-* Install sox and latex as above. 
-* Create a conda environment using `conda env create -f environment.yml`
-* **WINDOWS ONLY** Install `pyreadline` via `pip install pyreadline`. 
+* Install SoX and LaTeX as above.
+* Clone this repository as above: `git clone https://github.com/3b1b/manim.git`.
+* Navigate into the repository as above: `cd manim`.
+* Create a conda environment using `conda env create -f environment.yml`.
+* Activate the environment: `conda activate manim`.
+* Add the repo's directory to the Python path: `conda develop .`.
+* Test your setup: `python -m manim example_scenes.py SquareToCircle -pl`.
```

Comment:
I think it might be safer to just call `python` from an Anaconda environment. Perhaps it's something screwy in my setup, but for me, `python3` actually calls a version of python outside of the active environment. When in an environment, the environment's `python` takes priority over any other installed python; the same does not seem to be true of `python3`.